lib/sender.rb

```diff
@@ -0,0 +1,30 @@
+class Sender
+  require 'twilio-ruby'
+
+  attr_reader :twilio, :client, :time
+
+  ACCOUNT_SID = 'ACd319a65e921b7f3a98db79f2f6a2af51'
```

Comment:
Look into using Environment Variables to save this information locally and avoid pushing sensitive information up to GitHub. I would advise correcting to to remove the data from GitHub ASAP.